packages/gatsby/src/utils/webpack.config.js

```diff
@@ -379,9 +385,14 @@ module.exports = async (
           require.resolve(`@babel/runtime/package.json`)
         ),
         "core-js": path.dirname(require.resolve(`core-js/package.json`)),
-        "react-hot-loader": path.dirname(
-          require.resolve(`react-hot-loader/package.json`)
-        ),
+        // TODO: Remove entire block when we make fast-refresh the default
+        ...(process.env.HOT_LOADER !== `fast-refresh`
```

Comment:
does this matter? We won't load react-hot-loader when fast-refresh is enabled, right? I would keep the todo :)

Reply:
ah good point. i forgot how these worked.